kvmagent/kvmagent/plugins/storage_device.py

```
@@ -543,30 +647,31 @@ def check_iscsi_conf():
+                            iqn, login_portal, cmd.iscsiChapUserName))
                     bash.bash_o(
-                        'iscsiadm --mode node --targetname "%s" -p %s:%s --op=update --name node.session.auth.password --value=%s' % (
-                            iqn, cmd.iscsiServerIp, cmd.iscsiServerPort, linux.shellquote(cmd.iscsiChapUserPassword)))
-                timeout = self.get_iqn_login_timeout(iqn, cmd.iscsiServerIp, cmd.iscsiServerPort)
-                r, o, e = bash.bash_roe('timeout -s SIGKILL %s iscsiadm --mode node --targetname "%s" -p %s:%s --login' %
-                            (timeout, iqn, cmd.iscsiServerIp, cmd.iscsiServerPort))
-                wait_iscsi_mknode(cmd.iscsiServerIp, cmd.iscsiServerPort, iqn, e)
+                        'iscsiadm --mode node --targetname "%s" -p %s --op=update --name node.session.auth.password --value=%s' % (
+                            iqn, login_portal, linux.shellquote(cmd.iscsiChapUserPassword)))
+                timeout = self.get_iqn_login_timeout(iqn, login_server_ip, login_server_port)
+                r, o, e = bash.bash_roe('timeout -s SIGKILL %s iscsiadm --mode node --targetname "%s" -p %s --login' %
```

Ruff (0.15.18)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)